lib/platform/github/index.ts

```diff
@@ -432,13 +432,13 @@ export async function initRepo({
   );
   parsedEndpoint.pathname = config.repository + '.git';
   const url = URL.format(parsedEndpoint);
-  config.storage = new GitStorage();
-  await config.storage.initRepo({
+  await gitfs.initRepo({
     ...config,
     url,
     gitAuthorName: global.gitAuthor?.name,
     gitAuthorEmail: global.gitAuthor?.email,
   });
+  config.isGitInitialized = true;
```

Comment:
Can we avoid this state in the platforms and instead hold it in gitfs somehow?

Reply:
Yes, I thought about introducing `isInitialized()` function

Reply:
Isn't it just used for `cleanRepo()`, which is just used for testing? If so then would it work to push the logic into `gitfs`?